backend/app/api/endpoints/internal/bots.py

```
@@ -0,0 +1,181 @@
+    bot_name: str
+    bot_namespace: str
+    mcp_servers: dict[str, Any] = Field(
+        default_factory=dict, description="MCP servers configuration from Ghost CRD"
+    )
+
+
+@router.get("/{bot_name}/mcp", response_model=MCPServersResponse)
+async def get_bot_mcp_servers(
+    bot_name: str,
+    namespace: str = Query(default="default", description="Bot namespace"),
+    db: Session = Depends(get_db),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)